src/gpu/intel/ocl/tile_ops.h

```diff
@@ -557,27 +576,28 @@ DEF_BLOCK2D_LOAD_STORE(half, ushort, 16, 16, u16_m8k32v1, 32, 8)
         tile_store_block2d(t, ptr, m, n, m, offset_r, offset_c); \
     }
 
-#define DECLARE_2D_TILE_LOAD_PACKED_HALF(tile_type, sg, br, bc, nbr, nbc) \
-    __attribute__((overloadable)) void tile_load_packed_half(tile_type *t, \
-            const global half *ptr, int m, int n, int ld, int offset_r, \
+
+#define DECLARE_2D_TILE_LOAD_PACKED_VEC(tile_type, type, vector_type, sg, br, bc, nbr, nbc) \
+    __attribute__((overloadable)) void tile_load_packed_vec(tile_type *t, \
```

Comment:
To emphasize that it's for length-2 vectors:
```suggestion
    __attribute__((overloadable)) void tile_load_packed_vec2(tile_type *t, \
```